hunts/src/EditPuzzleModal.js

```diff
@@ -26,6 +27,10 @@ function EditPuzzleModal({ huntId, puzzleId, name, url, isMeta, hasChannels }) {
         },
       })
     ).finally(() => {
+      // Used to update hunt tags.
+      dispatch(fetchHunt(huntId));
```

Comment:
Same as above, we only need to do this if `isMeta` is different from `newIsMeta` right?